k8s/docker-entrypoint.sh

```diff
@@ -57,7 +57,8 @@ sleep $DEBUG_SLEEP
 if [ "$MEDUSA_MODE" == "RESTORE" ]; then
     restore
 elif [ "$MEDUSA_MODE" == "GRPC" ]; then
-    grpc
+    echo "Starting Medusa gRPC service"
+    exec python3 -m medusa.service.grpc.server server.py
```

Comment:
If changing the `grpc()` function above in this script has the desired affect, can we not simply call the function here (like what was done before)?

Also, would we need to invoke `exec` on line 44 [here](https://github.com/thelastpickle/cassandra-medusa/pull/313/files#diff-7e552c6093dd30777eac14ab0ab705b391b0d963d156ccda72178fef2b3514a6R44) as well, or is there no issue for restore?

Reply:
I need to clean this up. Provided the function call does not result in a new shell/process we should be able to leave it as is and just do the `exec` inside the `grpc` function. I will test.

>Also, would we need to invoke exec on line 44 here as well, or is there no issue for restore?

Initially I didn't think we needed it for restore since it is a long-live server process; however a restore operation could take a long time. I think we probably want it for restore as well to be on the safe side.